BUILDING.md

```diff
@@ -106,6 +106,7 @@ make generate
 >   * `no_btrfs`: A build tag disables building the Btrfs snapshot driver.
 >   * `no_devmapper`: A build tag disables building the device mapper snapshot driver.
 >   * `no_zfs`: A build tag disables building the ZFS snapshot driver.
+>   * `no_systemd`: Disable systemd notifications via [sd_notify](https://www.freedesktop.org/software/systemd/man/sd_notify.html) on Linux.
```

Comment:
it doesn't belong to snapshotters. We should remove the indent here

Reply:
Good catch.
Fixed.